copilot/fastapi-proxy/src/service/router.ts

```
@@ -0,0 +1,44 @@
+    }
+    delete headers.host; // Remove host header as it would be incorrect for the target
+
+    const response = await fetch(targetUrl, {
+      method: req.method,
+      headers,
+      body: req.body ? JSON.stringify(req.body) : null,
+    });
+
+    // Check if response is JSON before parsing
+    const contentType = response.headers.get('content-type');
+    if (contentType && contentType.includes('application/json')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)